packages/playground/assets/js/playground.js

```diff
@@ -66,11 +49,30 @@
 	});
 
 	await client.isReady();
-	await client.goTo('/');
 
-	// if (playground.pluginSlug) {
-	// 	await client.goTo('/wp-admin/plugins.php');
-	// } else {
-	// 	await client.goTo('/wp-admin/');
-	// }
+	// Login as the current user without a password
+	await client.writeFile(
+		'/wordpress/playground-login.php',
+		`<?php
+		require_once( dirname( __FILE__ ) . '/wp-load.php' );
+		if ( is_user_logged_in() ) {
```

Comment:
Just noting this makes it impossible to log out in the sandbox site. How about basing this on a cookie or a site option instead?

Reply:
Oh, sorry, you are right!